mlem/contrib/flyio/meta.py

```diff
@@ -110,6 +110,8 @@ def _create_app(self, state: FlyioAppState):
                 args["generate-name"] = True
             if self.get_env().access_token:
                 args["access-token"] = self.get_env().access_token
+            if self.org:
+                args["org"] = self.org
```

Comment:
I couldn't find any tests,

Reply:
Good point, looks like we didn't add them 👀

Reply:
Is this something you're interested in? As a first step I think we should add a test for `_create_app`, asserting that resulting `fly.toml` looks right. This test can also help us pin down the root cause of https://github.com/iterative/mlem/issues/657

The second step would be to add a real e2e deploy to fly. I can create a test account there and set up GHA workflow to make it work.

Reply:
Yeah sounds good. Do you want to add the tests in a separate PR?

Reply:
No, this PR is the right place :)

Reply:
@aguschin Any idea how this can be tested?
I'm doing
```py
def test_create_app():
    flyio_app = FlyioApp()
    state = flyio_app.get_state()
    flyio_app._create_app(state)
```

But the `get_state()` fails because of `mlem.core.errors.MlemObjectNotSavedError: Not saved object has no location`.
even if I create the app with a default state, while updating the state the issue happens again. I think I'm missing a step before the the call to create_app.

Reply:
Hi @aminalaee! Sure, let me help you out 🙌🏻 
It needs to be dumped first:
```py
from mlem.contrib.flyio.meta import FlyioApp


def test_create_app(tmpdir):
    flyio_app = FlyioApp()
    flyio_app.dump(str(tmpdir))
    state = flyio_app.get_state()
    flyio_app._create_app(state)
```

If you run this test, it will fail still, because creating flyio app requires Dockerfile present (`...Could not find a Dockerfile, nor detect a runtime or framework from source code. Continuing with a blank app.\x1b`).

Reply:
You either need to bypass this, or maybe call `mlem.api.deploy` directly on `flyio_app` while mocking some actual deployment mechanisms. Then, until the temp dir with `fly.toml` and stuff is removed, you can read it in the test to make some checks. Hope that makes it clearer! If confused, please ping me, I'll help with that.